### ISSUE TITLE
RC 4.0 Tint Color Not Working?

### ISSUE THREAD

**Issue by vtjon** [rhanb/nativescript-bottombar#79]
On iOS, with TNS 5.0, I can't seem to get the activeTintColor set. It remains blue. If I step through bottombar.ts, the getDefault/setNative properties for the activeTinitColor do not get called. The other color properties do get called. 

Any ideas? I can't seem to find anywhere that would cause this.

**Comment by rhanb** [comment]
@vtjon please provide a sample code and more details about your issue

**Comment by vtjon** [comment]
I think this covers it.
https://github.com/vtjon/bottombar-tint-color
![simulator screen shot - iphone 8 - 2018-12-10 at 12 25 31](https://user-images.githubusercontent.com/6945534/49749858-645e2780-fc77-11e8-8044-26ab8b6850ed.png)

Thanks for your help.

**Comment by rhanb** [comment]
@vtjon ,

Thank you for the repo, I manage to reproduce the issue. Basically it comes from the default value of the CSS Property `activeTintColor` which is `black`; but it doesn't seem to actually apply. So when you set the `activeTintColor` to a similar value to black example `#00000`, the `setNative` function will never be called since there is a `equalityComparer` callback which check if the new value of the property is different of the old one before calling the `setNative` function.

To summarise, the only issue there is, is the default value of the property which is not applied natively. The behavior of `equalityComparer` and the `setNative` functions are good.

Until I fix the issue, you can change the value of your `activeTintColor` property to a slightly different value like `#24292D` and it should work.

**Comment by rhanb** [comment]
After investigation, all the default values weren't working. Basically, if you just set the value of the different CSS properties to its default value, the values will never be applied.

This is not due to a wrong behavior of `equalityComparer` nor `setNative`. But a miss-conception of `defaultValue` that I assumed, setting a `defaultValue` to a property won't actually call the `setNative` to set the value natively. You have to do it programmatically when you instantiate the view, for example inside the `createNativeView` hook.

**Comment by rhanb** [comment]
@vtjon new version `4.0.0-rc1.2` published as the `next` version. Should fixed your issue. 

`tns plugin remove nativescript-bottombar && tns plugin add nativescript-bottombar@next`
